src/otx/algo/detection/heads/base_head.py

```diff
@@ -0,0 +1,454 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
```

Comment:
Does this file include only your own implementations or is it borrowed from somewhere?

Reply:
It's from mmdet

Reply:
You must obey this redistribution rule not to violate the license:
https://github.com/open-mmlab/mmdetection/blob/cfd5d3a985b0249de009b67d04f37263e11cdf3d/LICENSE#L91-L130
Contact @yunchu and take appropriate action.

Reply:
Licenses have been updated. Please check them.